x/wasm/keeper/proposal_handler.go

```diff
@@ -94,6 +95,7 @@ func handleInstantiateProposal(ctx sdk.Context, k types.ContractOpsKeeper, p typ
 		sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
 		sdk.NewAttribute(types.AttributeKeyCodeID, fmt.Sprintf("%d", p.CodeID)),
 		sdk.NewAttribute(types.AttributeKeyContract, contractAddr.String()),
+		sdk.NewAttribute(types.AttributeResultData, hex.EncodeToString(data)),
```

Comment:
Want to call this `types.AttributeResultDataHex`? to make it clear it is hex encoded?
That is what they are doing with ibc binary data (rather than just `string(rawBytes)` as before)

Reply:
Are these also returned in the non-proposal methods? 
(too lazy to open up the IDE... but as long as they are all the same, looks good to me)

Reply:
I have added this to the other places, too